### ISSUE TITLE
[BUG] Scrolling is very slow/unresponsive after prolonged usage

### ISSUE THREAD

**Issue by gphung** [eni9889/IG-PP-Issues#368]
iOS Version: iOS 11.4b3
Instagram Version: 61.0
IG ++ Tweak Version: 1.8r-187
Crash Report:
Jailbreak

After using IG for a few minutes using the stock-IG home view (i.e., not grid view), scrolling (swiping down/up) to navigate between posts become slow/unresponsive. Specifically, sometimes when I initiate the scrolling action, it takes a good half a second before the screen actually scrolls. With even more prolonged use, this scrolling response "delay" can be as long as a whole second or so.

Issue is not exclusive to Instagram 61.0 and IG++ 1.8r-187. I've noticed the issue since I jailbroke my iPhone X, which was when the iOS 11.4b3 signing window was still open.

**Comment by eni9889** [comment]
Hmm I've noticed the same thing. I'll take a look and see what I can find. Thanks for reminding me.

**Comment by hazanshoe** [comment]
Also having same issue with lagginess. I’m also on iPhone X iOS 11.4 beta 3 
Running newest version of Instagram as of 9/15

**Comment by tonynca** [comment]
Yeah me too. Have to close and open app for it to go back to normal
